hedera-node/configuration/compose/settings.txt

```diff
@@ -6,6 +6,8 @@ chatter.useChatter,                            false
 doUpnp,                                        false
 maxOutgoingSyncs,                              1                            # differs from mainnet
 numConnections,                                1000
+sync.syncAsProtocolEnabled,                    true
```

Comment:
At some point, we should also remove the `maxOutgoingSyncs` value. Not sure when that should be. Maybe after SAP is live on mainnet.

Reply:
@poulok I had a couple reasons for including it despite the default:
- make it more obvious downstream that a change has occurred
- make it very easy to toggle off via `settings.txt` in case we need to backpedal the feature

Would you prefer I omit settings that match the default configuration?

Reply:
That makes sense. Please write a ticket to clean this up at a later point, to include removing the redundant enable, and remove the maxOutgoingSyncs value

Reply:
https://github.com/hashgraph/hedera-services/issues/7279